docs/content_management/configure_ct_field_settings.md

```diff
@@ -28,15 +35,15 @@ general settings.
 |Translatable|Indicates whether a value of the Field can be translated.|Optional|
 |Can be a thumbnail|Indicates whether the Field can be a thumbnail.|Optional|
 
-## Default configuration of Pages
+## Default configuration of pages
 
 The following settings control the behavior of Content items of [Page](../content_management/create_edit_pages.md) type. 
 You modify them in the **Field definitions** section, the **Landing Page** Field.
 
 ### Block display
 
 You can define which page blocks are available to an editor in the page edit mode.
-You do it, for example, when a [developer creates a new block]([[= developer_doc =]]/content_management/pages/create_custom_page_block/) and you want to allow its selection.
+You do it, for example, after a [developer creates a new block]([[= developer_doc =]]/content_management/pages/create_custom_page_block/) and you want to allow adding it to the page.
```

Comment:
I'd rather avoid "useful". We plainly say "you do it" because that is what you have to do, otherwise the blosk is not available. And the developer is a handy tool to introduce active voice and explain why we redirect to the DG